src/main/java/seedu/address/logic/commands/CommandResult.java

```diff
@@ -21,31 +24,48 @@
      */
     private final boolean exit;
 
+    /**
+     * {@code view} command is executed
+     */
+    private final boolean view;
```

Comment:
i put this as view because there is a method named isView, not sure if it will cause any confusion. I thought it was okay because it aligns with the other two boolean variables, `exit` and `showHelp`. @channne Care to weigh in?

Reply:
Perhaps we should rename all of them 🤩

Reply:
ok 🤩